source/release-notes/v1.8-release-notes.rst

```diff
@@ -137,3 +143,23 @@ Environment variable name changes
 
 .. _dex: https://github.com/dexidp/dex
 .. _mod_auth_openidc: https://github.com/zmartzone/mod_auth_openidc
+
+Drop IE support
+...............
+
+IE support was officially dropped. See :ref:`Browser Requirements <browser-requirements>`.
+
+Native VNC connection tab for OOD can be enabled for sites external to OSC
+..........................................................................
+
+Added Sinatra gems into ondemand-gems for other apps to use
+...........................................................
+
+Specify the default login host in the cluster config
+....................................................
```

Comment:
This is done mostly in the customization.rst file. I think the "Shell hosts are now protected behind allowlists" sections link back to it.